projects/buck2/build.mk

```diff
@@ -0,0 +1,50 @@
+# --------------------------------------------------------------------------------
+# Project: Buck2 Build Script
+# Description: This script is used for building and managing the Buck2 project.
+# Author: Christian Abrokwa
+# --------------------------------------------------------------------------------
+
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+
+$(eval $(call project-define,buck2))
+
+PROJECT_NAME := buck2
+PROJECT_VERSION := 2023-09-01
+
+HOST_OUT_DIR := out/host/bin
+BUILD_HOST_DIR :=build/host
+
+BUCK2_HOST_OUT_TARGET := $(HOST_OUT_DIR)/buck2
+BUCK2_HOST_BUILD_TARGET := $(BUILD_HOST_DIR)
+BUCK2_ZIP_TARGET := $(BUCK2_OUT_TARGET).zip
+
+BUCK2_URL := https://github.com/facebook/$(PROJECT_NAME)/archive/refs/tags/$(PROJECT_VERSION).zip
+
+# ZSTD := $(shell command -v zstd 2> /dev/null)
```

Comment:
I see, we might need zstd to unpack buck2. Maybe for now let's just add it to the list of dependencies in this file: [scripts/jammy-install-deps.sh](https://github.com/facebookexperimental/ExtendedAndroidTools/blob/main/scripts/jammy-install-deps.sh). It will become available after the docker image is rebuilt.

Reply:
Alright, sure. I was thinking of adding it to the jammy-install-deps.sh file, but I was a bit hesitant to modify that one. Thanks.